site/content/zh/latest/tasks/quickstart.md

```diff
@@ -10,9 +10,9 @@ description: 只需几个简单的步骤即可开始使用 Envoy Gateway。
 
 一个 Kubernetes 集群。
 
-**注意：**请参考[兼容性表格](../install/matrix)来查看所支持的 Kubernetes 版本。
+**注意：** 请参考[兼容性表格](../install/matrix)来查看所支持的 Kubernetes 版本。
 
-**注意：**如果您的 Kubernetes 集群没有负载均衡器实现，我们建议安装一个
+**注意：** 如果您的 Kubernetes 集群没有负载均衡器实现，我们建议安装一个
```

Comment:
```suggestion
**注意：**如果您的 Kubernetes 集群没有负载均衡器实现，我们建议安装一个
```

Reply:
Same as https://github.com/envoyproxy/gateway/pull/3148#discussion_r1565371868

Reply:
![image](https://github.com/envoyproxy/gateway/assets/34599468/c26f17de-7880-4327-bc9f-d8a313ddc82a)

Because the the English version is that the ":" is bold.  Should we cancel the bold of ":" in Chinese?

Reply:
Of cause yes, I found no standards for this format, some of content with: `**Note:**`, and some are: `**Note**:`, let's make the standard in Chinese content first.

Reply:
![image](https://github.com/envoyproxy/gateway/assets/34599468/f8daff70-dd46-4170-ba42-9b809d5bda25)

Yes! I was confused about that there is no standards. But if we make the standard of bold the ":" or not, my suggestion is bold the ":". Because the ":" mark is the part of the "Note", not belong to the sentance. And I also found the other documentation such as kubernetes.io. They bold the colons, too.